draft-ietf-quic-transport.md

```diff
@@ -1834,7 +1834,12 @@ new connection IDs using the NEW_CONNECTION_ID frame.
 An endpoint might need to send packets on multiple networks without receiving
 any response from its peer.  To ensure that the endpoint is not linkable across
 each of these changes, a new connection ID is needed for each network.  To
-support this, multiple NEW_CONNECTION_ID messages are needed.
+support this, multiple NEW_CONNECTION_ID messages are needed.  Each
+NEW_CONNECTION_ID is marked with a sequence number.  Connection IDs MUST be used
+on only one local address, and each local address MUST advance to a connection
+ID with a later sequence number each time the connection ID changes.  Once a
+connection ID has been used, connection IDs with an earlier sequence number MUST
+NOT be used for packets with a greater packet number from that local address.
```

Comment:
I'm confused as to how this works with probing a new path (but not moving to it). Assume I am currently on Path A using CID 1, then I probe Path B with CID 2. This text seems to state that I may not use CID 1 on Path A with any packet number greater than I used to probe Path B with  CID 2. Is that correct?

Reply:
At that point, you may not use a CID < 1 on path A, and may not use a CID < 2 on path B.  You also aren't allowed to use CID = 2 on path A, because it's been used on a different path.

Reply:
Basically:
- For each path, the CID sequence number must advance, but the currently-in-use CID for each path is unrelated.  You could keep probing forever with CID2 on Path B even while Path A has moved up to CID 39, so long as the path isn't abandoned.
- Each CID can only be used on a single path.  If you used CID2 on Path B, you have to skip CID2 on Path A.

Reply:
Ok, maybe instead of "Once a connection ID has been used, connection IDs with an earlier sequence number MUST NOT be used for packets with a greater packet number from that local address." something like "Once a new connection ID is used, earlier connection IDs MUST NOT be used for new packets sent from the new local address".